playpen/ansible/roles/dev/tasks/debug.yml

```diff
@@ -0,0 +1,89 @@
+# All tasks in this file operate conditionally based on the truthiness of the pulp_dev_debug var
+---
+# Tools handy for debugging that aren't generally useful in normal development
+- name: Install debugging tools
+  dnf: name={{ item }} state=present
+  with_items:
+    - gdb
+    - lsof
+    - tcpdump
+
+# The crazy list of packages below was not generated by hand.
+# To generate it, first get all the debuginfo packages that gdb demands installed
+# Then, "rpm -qa | grep debuginfo | sed 's/debuginfo\(.*\)/debuginfo/' | sort"
+# The sed strips the EVRA off of the package NEVRA so it's not such a pain to maintain
+- name: Install debuginfo packages for gdb
+  dnf:
+    name: "{{ item }}"
+    state: present
+    enablerepo: fedora-debuginfo, updates-debuginfo
+  with_items:
+    - acl-debuginfo
+    - attr-debuginfo
+    - audit-debuginfo
+    - cracklib-debuginfo
+    - curl-debuginfo
+    - cyrus-sasl-debuginfo
+    - deltarpm-debuginfo
+    - e2fsprogs-debuginfo
+    - elfutils-debuginfo
+    - expat-debuginfo
+    - file-debuginfo
+    - freetype-debuginfo
+    - gdbm-debuginfo
+    - glib2-debuginfo
+    - glibc-debuginfo
+    - glibc-debuginfo
```

Comment:
I'll also add a `| uniq` to the generation command to prevent this from biting others...

This might indicate something a little worrisome, which is that the only way that package would show up on the list twice is if it's installed twice, which likely means there are two architectures installed...

I'll dig a little deeper, and good catch!